lib/locales/pagy.yml

```diff
@@ -156,12 +141,10 @@ tr:
         zero: "kayıt"
         one: "kayıt"
         other: "kayıt"
-    compact:
-      page: "Sayfa"
-      of: "/"
+    compact: "Sayfa %{page_input} / %{pages}"
     items:
-      show: "Göster"
-      items: "sayfa başı"
+      one: "Göster %{items_input} sayfa başı"
+      other: "Göster %{items_input} sayfa başı"
```

Comment:
```ruby
one: "Sayfada %{items_input} kayıt göster"
other: "Sayfada %{items_input} kayıt göster"
```

is grammatically more correct.

Reply:
Thank you!

Reply:
One question: is there any difference if the items in the `items_input` are 1 or more than 1?

Reply:
No, there is not.